azkaban-web-server/src/main/java/azkaban/imagemgmt/servlets/ImageVersionServlet.java

```diff
@@ -108,6 +113,26 @@ protected void handleGet(HttpServletRequest req, HttpServletResponse resp, Sessi
   @Override
   protected void handlePost(HttpServletRequest req, HttpServletResponse resp, Session session)
       throws ServletException, IOException {
+    final Map<String, String> templateVariableToValue = new HashMap<>();
+    log.info("URI Template: " + SINGLE_IMAGE_VERSION_URI_TEMPLATE.toString());
+    log.info("req uri: " + req.getRequestURI());
```

Comment:
Nit: Add detail which req URI?

Reply:
Removed the info log as they were added for my debug purpose. Not required as there is else block. All the invalid routes request will go to the else block.